doc/en/user/source/production/config.rst

```diff
@@ -331,6 +346,16 @@ If you wish to change this behavior you can do so through the following properti
 These properties can be set either via Java system property, command line argument (-D), environment
 variable or web.xml init parameter.
 
+.. _production_config_csp:
+
+Content-Security-Policy
+'''''''''''''''''''''''
+
+In order to mitigate cross-site scripting and clickjacking attacks GeoServer defaults to setting
+the Content-Security-Policy HTTP header based on rules loaded from a configuration file. See the
```

Comment:
For user's of the web admin console they do not see a configuration file.
```suggestion
the Content-Security-Policy HTTP header based on rules loaded from :ref:`security_csp` page.  See the
```

Reply:
I will remove the reference here to a configuration file.